static/src/javascripts/projects/commercial/sentinel.spec.ts

```diff
@@ -10,64 +10,144 @@ const config = (config_ as unknown) as {
 	get: jest.MockedFunction<(s: string, d: boolean) => boolean>;
 };
 
-jest.mock('lib/raven', () => ({
-	captureMessage: jest.fn(),
-}));
+MockDate.set(new Date('2021-01-01T00:00:00Z'));
+
+const CODE_ENDPOINT = '//logs.code.dev-guardianapis.com/log';
+const PROD_ENDPOINT = '//logs.guardianapis.com/log';
+const TEST_URL = 'http://testurl.theguardian.com/';
+
+const owner = 'commercial.sentinel';
+const mockDate = new Date();
+const defaultEvent: SentinelLoggingEvent = {
+	received_timestamp: mockDate,
+	received_date: mockDate.toISOString().slice(0, 10),
+	label: owner,
+};
 
 jest.mock('../../lib/config', () => ({
 	get: jest.fn(),
 }));
 
+beforeEach(() => {
+	navigator.sendBeacon = jest.fn();
+});
+
 afterEach(() => {
 	jest.clearAllMocks();
 });
 
+afterAll(() => {
+	MockDate.reset();
+});
+
 describe('sentinel', () => {
-	test('does not send a message when switches.sentinelLogger is false', () => {
+	test('should not send an event when switches.sentinelLogger is false', () => {
 		config.get.mockReturnValue(false);
-		amIUsed('moduleName', 'functioName');
-		expect(raven.captureMessage).not.toHaveBeenCalled();
+		amIUsed('moduleName', 'functionName');
+		expect(navigator.sendBeacon as jest.Mock).not.toHaveBeenCalled();
 	});
 
-	test('does send a message when switches.sentinelLogger is true', () => {
+	test('should send an event when switches.sentinelLogger is true', () => {
 		config.get.mockReturnValue(true);
-		amIUsed('moduleName', 'functioName');
-		expect(raven.captureMessage).toHaveBeenCalledTimes(1);
+		amIUsed('moduleName', 'functionName');
+		expect(navigator.sendBeacon as jest.Mock).toHaveBeenCalledTimes(1);
 	});
 
-	test('does not attach a label when it is not present', () => {
-		config.get.mockReturnValue(true);
+	test('should use the correct logging CODE endpoint', () => {
+		config.get.mockReturnValueOnce(true).mockReturnValueOnce(true); // first get checks switches.sentinelLogger, the second page.isDev
 		amIUsed('moduleName', 'functionName');
-		expect(raven.captureMessage).toHaveBeenCalledWith(
-			'moduleName.functionName',
-			expect.any(Object),
+		expect(navigator.sendBeacon as jest.Mock).toHaveBeenCalledWith(
+			CODE_ENDPOINT,
+			expect.any(String),
 		);
 	});
 
-	test('does attach a label when it is present', () => {
-		config.get.mockReturnValue(true);
-		amIUsed('moduleName', 'functionName', 'label=test');
-		expect(raven.captureMessage).toHaveBeenCalledWith(
-			'moduleName.functionName.label=test',
-			expect.any(Object),
+	test('should use the correct logging DEV endpoint', () => {
+		config.get.mockReturnValueOnce(true).mockReturnValueOnce(false); // first get checks switches.sentinelLogger, the second page.isDev
```

Comment:
You can chain commands across multiple lines.

```suggestion
		config.get
			.mockReturnValueOnce(true)  // for switches.sentinelLogger
			.mockReturnValueOnce(false); // for page.isDev
```

Reply:
I did try this but prettier complains if it's spread across multiple lines :(